packages/renderer/src/components/ExpandMore/component.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+import { Collapse, IconButton, Typography, Box } from '@mui/material';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { styled } from '@mui/material/styles';
+
+import type { Props } from './types';
+
+import './styles.css';
+
+const Arrow = styled((props: any) => {
```

Comment:
what's this sorcery ? can we use simple components instead.
it's really hard to follow